components/HomeContainer/ResultContainer/Result/ItemName.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Link from "next/link";
+
+const ItemName = ({ name = "", url = "" }) => {
+  return (
```

Comment:
We should be explicit about the interface rather than depending on type inference.

```suggestion
interface Props{
   name: string,
   url: string
}

const ItemName = ({ name, url }: Props) => {
  return (
```